src/python/pants/option/options_test.py

```diff
@@ -1046,7 +1046,7 @@ def test_shadowing() -> None:
         args=["./pants"],
     )
     options.register("", "--opt1")
-    options.register("foo", "-o", "--opt2")
+    options.register("foo", "--opt2")
```

Comment:
This was a test that didn't adhere to our own validation rules, but skirted them since we don't parse in this test.